content/blog/2019/07/2019-07-04-use-casc-for-exws.adoc

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Use Terraform with External Workspace Manager to Support EFS"
+tags:
+- jenkins
+- gsoc
+author: yufei_zhang
+description: "A brief example of using Terraform with EFS" # optional
+opengraph:
+---
+
+=== Why you want to use External Workspace Manager with EFS
+
+What is Terraform : A CasC software, like CloudFormation and Ansible designed for managing cloud resources. It can free you from managing cloud resource by console, and can be version controlled using git as it is configuration-as-code. Currently it supports AWS and Azure best. AWS CloudFormation is for AWS only.
```

Comment:
```suggestion
What is Terraform : A configuration management software, like CloudFormation and Ansible designed for managing cloud resources. It can free you from managing cloud resource by console, and can be version controlled using git as it is configuration-as-code. Currently it supports AWS and Azure best. AWS CloudFormation is for AWS only.
```